package.json

```diff
@@ -2,11 +2,11 @@
   "name": "dequal",
   "version": "1.0.1",
   "repository": "lukeed/dequal",
-  "description": "A tiny (305B) utility for check for deep equality",
-  "unpkg": "dist/dequal.min.js",
-  "module": "dist/dequal.mjs",
-  "main": "dist/dequal.js",
-  "types": "dequal.d.ts",
+  "description": "A tiny (304B) utility for check for deep equality",
+  "unpkg": "dist/index.min.js",
```

Comment:
Why not keep everything named?

Reply:
CommonJS users/old Node.js instances can `require('dequal/lite')` without needing a direct link.